content/concept-history-sharding.md

```diff
@@ -0,0 +1,56 @@
+# History Sharding
+
+As servers operate, they naturally produce a database containing the nodes from the ledgers they witnessed or acquired during network runtime. This ledger data is stored by the node store, but the online delete logic rotates these databases when their size exceeds configured space limitations.
+
+Rippled sharding is a feature that manages storing and sharing of distributed historical data as shards, which individually contain data for a segment of ledger history and are stored separately, in a shard store. In this way, Rippled sharding provides a middle ground between running a rippled server with full-history and one with only recent ledger data. A shard store does not replace a node store, but implements a reliable path towards distributed ledger history across the XRP Ledger Network.
+
+[![XRP Ledger Network: Nodes and Shard Store Diagram](img/xrp-ledger-network-node-and-shard-stores.png)](img/xrp-ledger-network-node-and-shard-stores.png)
+
+## Acquiring and Sharing Shards
+
+Acquiring shards begins after synchronizing with the network and performing ledger backfilling. During this time of lower network activity, a `rippled` server set to maintain a `shard_db` randomly chooses a shard to add to its shard store. To increase the probability for an even distribution of the network ledger history, shards are randomly selected for acquisition, and the current shard is given no special consideration.
```

Comment:
What exactly is a "shard" anyways? A leaf node? A full SHAmap? A range of ledgers with all their objects + InnerNodes?

Sounds like the latter ("a range of ledgers and all their objects").

Reply:
@MarkusTeufelberger Yes, a shard is a range of ledgers stored in a database. Ledgers are stored in the node or shard store exactly the same. A ledger is comprised of a header and two SHAMaps.